ocean-client/src/programs/common-program.ts

```diff
@@ -143,25 +143,18 @@ export class CommonProgram {
 
     }
 
-    async depositToVault(symbol: string, amount: BigNumber): Promise<boolean> {
-        const token = await this.getTokenBalance(symbol)
-        if (!token) {
-            return false
-        }
-        const address = await this.account!.getAddress()
-        console.log("depositToVault vaultId=" + this.settings.vault + " from=" + address + " token=" + amount + " " + token.symbol)
```

Comment:
don't we need a log output for such events?

Reply:
nvm found it, the caller does the log output

Reply:
yep, all the other calls don't log themself either, so I went for consistency. wouldn't mind moving it into the calls in general thou